tests/common/helpers/drop_counters/drop_counters.py

```diff
@@ -0,0 +1,104 @@
+import logging
+import re
+import json
+import pytest
+from tests.common.utilities import wait_until
+
+logger = logging.getLogger(__name__)
+
+# CLI commands to obtain drop counters.
+NAMESPACE_PREFIX = "sudo ip netns exec {} "
+NAMESPACE_SUFFIX = "-n {} "
+GET_L2_COUNTERS = "portstat -j "
+GET_L3_COUNTERS = "intfstat -j "
+ACL_COUNTERS_UPDATE_INTERVAL = 10
+LOG_EXPECT_ACL_RULE_CREATE_RE = ".*Successfully created ACL rule.*"
+LOG_EXPECT_ACL_RULE_REMOVE_RE = ".*Successfully deleted ACL rule.*"
+LOG_EXPECT_PORT_ADMIN_DOWN_RE = ".*Configure {} admin status to down.*"
+LOG_EXPECT_PORT_ADMIN_UP_RE = ".*Port {} oper state set from down to up.*"
+RX_DRP = "RX_DRP"
+RX_ERR = "RX_ERR"
+
+PKT_NUMBER = 1000
```

Comment:
I don't know how relevant this constant is to the dualtor testing - maybe it should stay in the `tests/drop_counters` folder? Should the below methods be updated to accept the number of expected drops as an argument?

Reply:
Agree. Methods in this script are updated to expect the packets count as an argument.
Updated the `test_drop_counters` also accordingly.